conf/nginx.conf

```diff
@@ -20,6 +20,36 @@ server {
     add_header X-XSS-Protection "1; mode=block" always; 
     add_header Referrer-Policy "no-referrer" always;
     
+    location /base58 {
+        if ($args = "go-get=1") {
+           add_header Content-Type text/html;
+           return 200 '<meta name="go-import" content="decred.org/base58 git https://github.com/decred/base58.git">';
+        }
+    }
+    location /cspp {
+        if ($args = "go-get=1") {
+           add_header Content-Type text/html;
+           return 200 '<meta name="go-import" content="decred.org/cspp git https://github.com/decred/cspp.git">';
+        }
+    }
+    location /dcrd {
+        if ($args = "go-get=1") {
```

Comment:
in case other args are present, perhaps match with `~`

Reply:
Actually, there's a `$arg_{name}` variable.  https://nginx.org/en/docs/http/ngx_http_core_module.html#var_arg_

Not sure how it works in practice though.